packit/cli/builds/obs_build.py

```diff
@@ -0,0 +1,261 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+import os.path
+import xml.etree.ElementTree as ET
+from dataclasses import dataclass
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Optional
+
+import click
+from osc import conf, core
+
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception, get_packit_api, iterate_packages
+from packit.config import get_context_settings, pass_config
+from packit.config.aliases import DEPRECATED_TARGET_MAP
+from packit.config.config import Config
+from packit.config.package_config import PackageConfig
+from packit.constants import (
+    PACKAGE_LONG_OPTION,
+    PACKAGE_OPTION_HELP,
+    PACKAGE_SHORT_OPTION,
+)
+
+logger = logging.getLogger(__name__)
+
+conf.get_config()
```

Comment:
This should be called in the click cli function, instead of globally. Can make a simple wrapper if you want to expose it outside the CLI.